test-data/unit/check-errorcodes.test

```diff
@@ -455,7 +455,7 @@ class E(TypedDict):
     y: int
 
 a: D = {'x': ''}  # E: Incompatible types (expression has type "str", TypedDict item "x" has type "int")  [typeddict-item]
-b: D = {'y': ''}  # E: Extra key "y" for TypedDict "D"  [typeddict-item]
+b: D = {'y': ''}  # E: Missing key "x" for TypedDict "D"  [typeddict-item] # E: Extra key "y" for TypedDict "D"  [typeddict-unknown-key]
```

Comment:
Could you please add a test verifying that _values_ for known keys are still type-checked if unknown keys are present?

Reply:
Good idea, will add it over the weekend

Reply:
Done!

I left them on the 'check-errorcodes' file but can come them to the typeddict specific file.